README.md

```diff
@@ -0,0 +1,233 @@
+<!--
+---
+name: Azure Functions TypeScript Event Grid Blob Trigger using Azure Developer CLI
+description: This template repository contains an Azure Functions reference sample using the Blob trigger with Event Grid source type, written in TypeScript (v4 programming model) and deployed to Azure using the Azure Developer CLI (azd). The sample uses managed identity and a virtual network to make sure deployment is secure by default.
+page_type: sample
+products:
+- azure
+- azure-functions
+- azure-blob-storage
+- azure-virtual-networks
+- entra-id
+urlFragment: functions-quickstart-typescript-azd-eventgrid-blob
+languages:
+- typescript
+- javascript
+- nodejs
+- bicep
+- azdeveloper
+---
+-->
+
+# Azure Functions TypeScript Event Grid Blob Trigger using Azure Developer CLI
+
+This template repository contains an Azure Functions reference sample using the Blob trigger with Event Grid source type, written in TypeScript (v4 programming model) and deployed to Azure using the Azure Developer CLI (`azd`). When deployed to Azure the sample uses managed identity and a virtual network to make sure deployment is secure by default. You can control whether a VNet is used in the sample by setting `VNET_ENABLED` to true or false in the AZD parameters.
+
+This sample implements a simple function that copies PDF files from an `unprocessed-pdf` container to a `processed-pdf` container when new blobs are created. This straightforward example showcases how to use the Event Grid blob trigger to automatically respond to blob creation events in near real-time.
+
+![Architecture diagram for Azure Functions Event Grid Blob Trigger](./img/architecture.png)
```

Comment:
You can copy the ./img folder from the referenced sample https://github.com/Azure-Samples/functions-quickstart-dotnet-azd-eventgrid-blob/ to make this image reference work